static/main.js

```diff
@@ -76,6 +77,70 @@ require("monaco-loader")().then(function () {
         });
     }
 
+    function setupButtons() {
+        var alertSystem = new Alert();
+
+        window.cookieconsent.initialise({
+            palette: {
+                popup: {
+                    background: "#eaf7f7",
+                    text: "#5c7291"
+                },
+                button: {
+                    background: "#56cbdb",
+                    text: "#ffffff"
+                }
+            },
+            theme: "edgeless",
+            type: "opt-in",
+            content: {
+                href: "#",
+                message: "Compiler Explorer uses cookies & related technologies to provide a better service.",
+                dismiss: 'Do NOT allow cookies',
+                header: 'This is amazing, isnt it?'
+            },
+            law: {
+                regionalLaw: false
+            },
+            location: {
+                services: []
+            },
+            cookie: {
+                domain: '.localhost:10240'
```

Comment:
Woops, testing value. It needs to be changed. Should we set it as a configurable parameter somewhere for 3rd party users? If so, is it secure to do so?